benchmarks/pytorch-lightning/README.md

```diff
@@ -0,0 +1,4 @@
+
+# Benchmark
+
+Rewrite this README to explain what the benchmark is!
```

Comment:
Doesn't have to be much, but describing the overall benchmark in a few lines would be handy.

Reply:
Fixed in 47085da